src/sympc/tensor/mpc_tensor.py

```diff
@@ -769,8 +770,12 @@ def truncate(
             result (MPCTensor): Truncated result
         """
         # Truncation should be added for RSTensor
+        from sympc.protocol import ABY3
```

Comment:
Q: This is the same case: if you move them at the top of the module they don't work?

Reply:
Yeah,we get a circular import error.